prob2_silver.cpp

```diff
@@ -0,0 +1,75 @@
+// Name: Ben Wei
+// 40 mins spent
+// Tests: all correct
+// Feedback: The "seniority" in this problem made me think of priority queue instantly.
+// From there, the coding wasn't super complicated so I mainly just tried to solve this
+// problem as fast as I could.
+
+#include <bits/stdc++.h>
+using namespace std;
+
+void setIO(string name = "") { // name is nonempty for USACO file I/O
+    ios_base::sync_with_stdio(0); cin.tie(0); // see Fast Input & Output
+    // alternatively, cin.tie(0)->sync_with_stdio(0);
+    if (name.size() > 0) {
+        freopen((name+".in").c_str(), "r", stdin); // see Input & Output
+        freopen((name+".out").c_str(), "w", stdout);
+    }
+}
+
+bool comp(const array<int, 3> &a, const array<int, 3> &b) {
+    return a[0] < b[0];
+}
+
+
+
+int main() {
+    setIO("convention2");
+    int n; cin >> n;
```

Comment:
Use capital letter or longer words for variables that scope over a few blocks. Use lower case letters for variables within a scope of less than 10 lines